test/lib/actions/ConfigAction.test.ts

```diff
@@ -252,7 +249,7 @@ describe('ConfigAction tests', () => {
 			it.each([
 				{prop: 'config_root'},
 				{prop: 'log_folder'}
-			])(`When derivable property $prop input is non-null and non-default, it is rendered as-is with no comment`, async ({prop}) => {
+			])(`When derivable property $prop input is non-null and non-default, it is rendered as-is`, async ({prop}) => {
```

Comment:
This test description was incorrect. There is a comment - which is "#modified from: <default>". So fixed it.